draft-ietf-tls-wkech.xml

```diff
@@ -673,6 +673,10 @@ Client <----------->|                    | facing
                 </li>
             </ul>
         </t>
+	<t>
+	    Operators should be careful not to put confidential information,
+	    such as private key material, in the .well-known URL.
```

Comment:
```suggestion
	    Operators should be careful to avoid directory enumeration or traversal attacks
	    if the .well-known/ URLs are served directly from a filesystem.  For example, a
	    misconfiguration of this kind could expose the ECH private keys.
```